C7/MainMenu.cs

```diff
@@ -1,10 +1,11 @@
 using Godot;
-using ConvertCiv3Media;
-using C7GameData;
 using System;
+using Serilog;
 
 public class MainMenu : Node2D
 {
+	static ILogger log = LogManager.ForContext<MainMenu>();
```

Comment:
@QuintillusCFC commented:
"Question: Is it possible to selectively enable logs?

For reference, I see it prints out:

12:37:32.429 [DBG] MainMenuMusicPlayer: setting volume to 100, which is 0 decibel offset

MainMenuMusicPlayer being the ForContext of that log.

I know some libraries, such as Log4J, have a hierarchical structure, where a fully-qualified class name is considered, such as com.civfanatics.xplatformeditor.map.rightSidePanel . This then allows turning on debug messages based on partially-qualified class name, such as "com.civfanatics.xplatformeditor.map" to turn on debugging statements for the map, but not for the whole application. I've only sparingly used that in practice, but it could be useful for debugging areas such as the graphics or AI, especially for hard-to-replicate events.

I see that we aren't using namespaces in the sample classes where it's enabled... so maybe the answer is that if we were it would be supported."

@TinaFemea replied:
"I totally agree with this. Having some way to show only the debug messages you're interested in is crucial."

@pcen replied:
"You can configure Serilog to filter by by namespace [like this](https://stackoverflow.com/a/38785632). With `ForContext`, the fully qualified class name is added to the log, but I believe it is just omitted from console output for brevity"